common/vgrepo.spec

```diff
@@ -96,12 +100,16 @@ rm -rf %{buildroot}
 
 %files
 %defattr(-,root,root,-)
-%config(noreplace) %{_sysconfdir}/%{name}.knf
+%config(noreplace) %{_sysconfdir}/%{name}/%{name}.knf
+%config(noreplace) %{_sysconfdir}/%{name}/templates/default.tpl
```

Comment:
Don't use `%config(noreplace)` for template. It used by default and better to update it to the latest version. If a user wants to change something, he can make a copy of this file.